jib-cli/src/main/java/com/google/cloud/tools/jib/cli/cli2/logging/CliLogger.java

```diff
@@ -22,36 +22,54 @@
 import com.google.common.annotations.VisibleForTesting;
 import java.io.PrintStream;
 
-/** A simple cli logger that logs to the command line based on the configured log level. */
+/** A simple CLI logger that logs to the command line based on the configured log level. */
 public class CliLogger {
 
   /**
-   * Create a new logger for the cli.
+   * Create a new logger for the CLI.
    *
    * @param verbosity the configure verbosity
    * @param consoleOutput the configured consoleOutput format
    * @return a new ConsoleLogger instance
    */
   public static ConsoleLogger newLogger(Verbosity verbosity, ConsoleOutput consoleOutput) {
-    CliLogger cliLogger = new CliLogger(verbosity, System.out, System.err);
-    boolean isRichConsole = isRichConsole(consoleOutput);
-
-    return newLogger(cliLogger, isRichConsole, new SingleThreadedExecutor());
+    return newLogger(
+        verbosity, consoleOutput, System.out, System.err, new SingleThreadedExecutor());
   }
 
   @VisibleForTesting
   static ConsoleLogger newLogger(
-      CliLogger cliLogger, boolean isRichConsole, SingleThreadedExecutor executor) {
-    // rich logger will use an explicit progress event handler
+      Verbosity verbosity,
+      ConsoleOutput consoleOutput,
+      PrintStream stdout,
+      PrintStream stderr,
+      SingleThreadedExecutor executor) {
+    boolean enableRichProgress =
+        isRichConsole(consoleOutput) && verbosity.atLeast(Verbosity.lifecycle);
     ConsoleLoggerBuilder builder =
-        isRichConsole
-            ? ConsoleLoggerBuilder.rich(executor, true)
-            : ConsoleLoggerBuilder.plain(executor).progress(cliLogger::lifecycle);
-    builder.error(cliLogger::error);
-    builder.warn(cliLogger::warn);
-    builder.lifecycle(cliLogger::lifecycle);
-    builder.info(cliLogger::info);
-    builder.debug(cliLogger::debug);
+        enableRichProgress
+            ? ConsoleLoggerBuilder.rich(executor, false)
```

Comment:
Flipping `enableTwoCursorUpJump` doesn't actually change output and is not required to fix the issue. However, since there's no need to employ the two-cursor-up-jump trick (was only necessary for Maven), I'm turning it off.